gpt_lw/nn.py

```diff
@@ -4,40 +4,40 @@
 from cloudpickle import cloudpickle
```

Comment:
why is this named nn.py? "model_utils.py" makes sense to me.

Reply:
I guess that is because in ZDC repo there was an `utils` folder and this file was inside as neural network utils ;) But now it can be changed